capplets/appearance/theme-installer.c

```diff
@@ -52,18 +56,18 @@ enum {
 	DIRECTORY
 };
 
-static gboolean
-cleanup_tmp_dir (GIOSchedulerJob *job,
-		 GCancellable *cancellable,
-		 const gchar *tmp_dir)
+static void
+cleanup_tmp_dir (GTask        *task,
+                 gpointer      source_object,
+                 gpointer      task_tada,
+                 GCancellable *cancellable)
```

Comment:
Build warnings `-Wunused-parameter`.
https://app.travis-ci.com/github/mate-desktop/mate-control-center/jobs/612083277#L4943
```
theme-installer.c:60:32: warning: unused parameter 'task' [-Wunused-parameter]

cleanup_tmp_dir (GTask        *task,

                               ^

theme-installer.c:62:32: warning: unused parameter 'task_tada' [-Wunused-parameter]

                 gpointer      task_tada,

                               ^

theme-installer.c:63:32: warning: unused parameter 'cancellable' [-Wunused-parameter]

                 GCancellable *cancellable)

                               ^
```

Reply:
```unused``` modifier variable needs to be added

Reply:
done

Reply:
Warnings are gone.